test/e2e/scalability/load.go

```diff
@@ -703,3 +720,30 @@ func CreateNamespaces(f *framework.Framework, namespaceCount int, namePrefix str
 	}
 	return namespaces, nil
 }
+
+func CreateQuotas(f *framework.Framework, namespaces []*v1.Namespace, podCount int, testPhase *timer.Phase) error {
+	defer testPhase.End()
+	quotaTemplate := &v1.ResourceQuota{
+		Spec: v1.ResourceQuotaSpec{
+			Hard: v1.ResourceList{"pods": *resource.NewQuantity(int64(podCount), resource.DecimalSI)},
+		},
+	}
+
+	for _, ns := range namespaces {
+		var got *v1.ResourceQuota
+		if err := wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			var err error
+			quotaTemplate.Name = ns.Name + "-quota"
+			got, err = f.ClientSet.CoreV1().ResourceQuotas(ns.Name).Create(quotaTemplate)
+			if err != nil {
+				framework.Logf("Unexpected error while creating resource quota: %v", err)
```

Comment:
I think you shouldn't be retrying failures unconditionally for the CREATE request. There are some non-retriable errors (i.e AlreadyExists).

Some earlier discussion about this -
 https://github.com/kubernetes/kubernetes/issues/55860#issuecomment-346039467

Reply:
Added a check for AlreadyExists.